### DISCUSSION TITLE
How do I localize an asset such as a PNG?

### DISCUSSION THREAD

**Discussion by kristofferwitt** [facebook/docusaurus#5294] [Q&A] [answered]
I have some PNGs with text on them that need to be translated.

What is the strategy for getting the i18n system to use the translated image instead of the default one? Relative paths? Colocation?

**Comment by slorber** [comment, marked as answer]
You can colocate the asset near the doc using it and use a relative path. 
The i18n folder could contain the localized asset and localized doc, still linked through relative paths (so that the path doesn't need to be updated in the localized doc).
If you use Crowdin, you can configure it to upload the asset to localize so that their UI allows to localize that asset.

In practice, I suspect this strategy has not been widely used so far so let me know if it doesn't work.

**Reply by kristofferwitt** [reply]
Okay, I will try it out, thanks!
